packages/obytelib/obytelib.1.6/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+authors: ["Benoît Vaugon"]
+homepage: "https://github.com/bvaugon/obytelib"
+bug-reports: "https://github.com/bvaugon/obytelib/issues"
+dev-repo: "git+https://github.com/bvaugon/obytelib.git"
+maintainer: "benoit.vaugon@gmail.com"
+license: "CeCILL-2.0"
+build: [
+  ["./configure" "-prefix" prefix]
+  [make "all"]
+]
+remove: [
+  ["./configure" "-prefix" prefix]
+  [make "uninstall"]
+]
+depends: [
+  "ocaml" {>= "4.07.0" & <= "4.14.0"}
```

Comment:
`<=` should be avoided as it eliminates minor releases
```suggestion
  "ocaml" {>= "4.07.0" & < "4.15.0"}
```

Reply:
Also removing the upper-boound entierly is also a possibility unless you know it will break with OCaml 5

Reply:
I am pretty sure that it will be broken in the next major release since bytecode version now change at all major releases. So I accept your first suggestion (< "4.15.0").